### DISCUSSION TITLE
confession

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#395] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i ate the hot cheeto my friend dont know pls no tell him!!

**Comment by Aka-but-nice** [comment, marked as answer]
Alr your secret is safe with me

**Comment by 3kh0** [comment]
this is gonna be added as a splash text some day

**Comment by Calamitou** [comment]
i alr told him X-X
